ports/stm32/mpconfigboard_common.h

```diff
@@ -394,6 +394,15 @@
 #define MICROPY_HW_RFCORE_BLE_LL_ONLY                   (1) // use LL only, we provide the rest of the BLE stack
 #endif
 
+// Configuration for STM32G4 series
+#elif defined(STM32G4)
+
+#define MP_HAL_UNIQUE_ID_ADDRESS (UID_BASE)
+#define PYB_EXTI_NUM_VECTORS (42) // up to 42 event/interrupt requests: 28 configurable lines, 14 direct lines
+#define MICROPY_HW_MAX_I2C (3)
+#define MICROPY_HW_MAX_TIMER (20) // TIM 1-8, 20
+#define MICROPY_HW_MAX_UART (5) // uart 1 - 5 + lpuart1
```

Comment:
@H-Grobben I'm wondering if this should be 4, not 5.  LPUART1 is excluded from this count, I think.  Eg there is the following:
```
struct _pyb_uart_obj_t *pyb_uart_obj_all[MICROPY_HW_MAX_UART + MICROPY_HW_MAX_LPUART];
```

Reply:
It depends on the specific type. There are types with 3 USARTS (and LPUART) only, and the added Nucleo type (G474Rx) has 3 USARTS and 2 UARTS (and LPUART). I was in doubt on the number I should include here, but my decision was the number of peripherals in the Nucleo board, so that's why it was 5.

Reply:
Ah, I see, thanks!  So it is correct as is, it should be 5 because that is the maximum possible on the G4 series.